lib/Controller/RoomController.php

```diff
@@ -1811,4 +1811,22 @@ public function setSIPEnabled(int $state): DataResponse {
 
 		return new DataResponse($this->formatRoomV2andV3($this->room, $this->participant));
 	}
+
+	/**
+	 * @NoAdminRequired
+	 * @RequireModeratorParticipant
+	 *
+	 * @return DataResponse
+	 */
+	public function resendEmails(): DataResponse {
+		$participants = $this->participantService->getParticipantsForRoom($this->room);
+		foreach ($participants as $participant) {
+			if ($participant->getAttendee()->getActorType() === Attendee::ACTOR_EMAILS) {
```

Comment:
should I also check if participantType is GUEST or is this safe already ?

Reply:
participant type could be guest or guest_moderator

Reply:
So no, dont

Reply:
indeed!